### ISSUE TITLE
Test Log In Page Interactive Elements for Accessibility

### ISSUE THREAD

**Issue by bpmcneilly** [internetarchive/openlibrary#4909]
# Page to Test:

Log In - https://openlibrary.org/account/login

# Steps

Steps for this task are outlined in #4897
 
# Expectations

Test the specified page according to the steps outlined in the parent issue. Create individual bugs with steps to reproduce for any found violations on this page. Reference those bugs on this tracking issue.

Once the page has been tested & all bugs are found, close this issue.

**Comment by Sabreen-Parveen** [comment]
@bpmcneilly I would like on this page.

**Comment by dbouris** [comment]
@bpmcneilly after checking the page for accessibility issues together with @Philippos01 we found that the login button at the bottom does not get focus. We opened issue  #6399 and plan to deal with it as we believe it is a css concerned one.
